cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
Ok but the another issue is we are setting private '_properties_form' to something that potentially isnt a properties form at all. Can we have _properties_form and _properties_region where the properties_form property would pick one, depending on what is available?